webpack.config.js

```diff
@@ -11,10 +11,6 @@ module.exports = {
             test: /\.js?$/,
             exclude: /node_modules/,
             loader: 'babel-loader',
```

Comment:
czy przecinek na końcu linii jest potrzebny?

Reply:
Rozumiem, że chodzi o sytuację, kiedy jest to ostatnia linia wewnątrz obiektu albo arraya. 
W takiej sytuacji w ES6 przecinek jest opcjonalny, ale ja go stawiam, bo potem jak w kolejnym komicie dokładam kolejną linię to w diffie widzę tylko dodanie nowej linii a nie zmianę tej ostatniej przez dodanie przecinka.

Zobacz róznicę tu https://imgur.com/a/UQBxeXX (oba obrazki pokazują dodanie tej samej jednej linii)

W JSON i w ES5 dodanie przecinka po ostatnim elemencie/linii w obiekcie/arrayu jest błędem, dlatego w ES6 to zmieniono, żeby dało się tak robić.

Reply:
No rzeczywiście. Przekonałeś mnie ;)